### DISCUSSION TITLE
3D溜溜网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#86] [模板讨论]
3D溜溜网 模板问题反馈评论区
有问题可以下面反馈。

**Comment by icefinger16** [comment]
不知道是不是66改版了，目前签到领不到下载币之类的了，只是登录了网站一下

**Reply by wjf0214** [reply]
![image](https://user-images.githubusercontent.com/23102821/228715583-80ef9738-bc8a-4ba9-86e6-6f8c02fa5782.png)
可以啊。我看我一直OK。

**Reply by icefinger16** [reply]
明白了，他有两个签到，登录网页有一个奖励（自动签到拿的就是这个），然后点你头像，有个用户签到也有奖励。

**Reply by wjf0214** [reply]
记不太清了，有点久远了。
不过我做的模板是有2个签到的。应该是你说的这个。。。看正常在跑就没管了。

**Comment by miaoxiaozhao** [comment]
Failed at 3/4 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://user.3d66.com/

**Reply by WernerHai** [reply]
https://user.3d66.com/ 替换成新的个人用户页就行https://user.3d66.com/newUser/homepage
